tests/support/__init__.py

```diff
@@ -296,12 +306,24 @@ def fixturefile(relative_path, fixture_format=None):
     #_, extension = os.path.splitext(os.path.basename(tmp_path))
     #assert fixture_format == extension, "fixture file does not match format"
 
-    if fixture_format == 'json':
-        return _fixturefile_json(tmp_path)
+    data = None
+
+    if fixture_format == 'binary':
+        with open(tmp_path, 'rb') as binfile:
+            data = binfile.read()
+    elif fixture_format == 'json':
+        data = _fixturefile_json(tmp_path)
     else:
         raise AssertionError(
             "unsupported fixture format: %s" % (fixture_format,))
 
+    return (data, tmp_path) if include_path else data
+
+
+def fixturefile_normname(relative_path, prefix=None):
+    normname, _ = relative_path.split('--')
+    return os.path.join(prefix, normname) if prefix else normname
```

Comment:
On a pedantic note I'm generally not too fond of these one-liner conditionals as they tend to save little and make code less readable IMO. E.g in this case it could easily hide details like if you really meant to test `prefix` to be logically `True` or actually wanted to only target the default value with `is not None`. Here `prefix=''` would not really make a difference, but for the general sake it might.